CMakeLists.txt

```diff
@@ -111,8 +111,6 @@ if(MSVC)
 		CMAKE_C_FLAGS_MINSIZEREL
 		CMAKE_C_FLAGS_RELWITHDEBINFO
 	)
-		# force /MD runtime (future note: cmake 3.15 has CMAKE_MSVC_RUNTIME_LIBRARY)
```

Comment:
Is this replaced by anything, or is it just deemed unnecessary?

Reply:
It is replaced by [policy CMP0091](https://cmake.org/cmake/help/latest/policy/CMP0091.html#policy:CMP0091) which is automatically enabled by setting the minimum required version to 3.18. If we don't set MSVC_RUNTIME_LIBRARY CMake defaults to MultiThreaded$<$<CONFIG:Debug>:Debug>DLL ( /MD or /MDd for debug builds).